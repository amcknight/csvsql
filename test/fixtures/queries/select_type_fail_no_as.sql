SELECT 2 + 3
FROM test/fixtures/tables/people
