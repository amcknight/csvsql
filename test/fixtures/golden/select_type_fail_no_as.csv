SQL Query Parser:1:10:
  |
1 | SELECT 2 + 3
  |          ^^^^
unexpected "+ 3<newline>"
expecting "FROM", ',', or white space
